package.json

```diff
@@ -22,13 +22,8 @@
     "@typescript-eslint/eslint-plugin": "^5.3.0",
     "@wordpress/api-fetch": "^5.2.4",
     "@wordpress/base-styles": "^4.0.0",
```

Comment:
Usually base-styles and api-fetch are also normal dependencies, unless something is different about their use here

Reply:
Good catch, updated in e45395de55831a59bdb378f6462e86e4a3387d51